### DISCUSSION TITLE
Salt Install Guide Overhaul, repo migration, and salt-bootstrap updates

### DISCUSSION THREAD

**Discussion by ScriptAutomate** [saltstack/salt#67027] [Announcements]
The Salt Install Guide has been updated to reflect the Salt package repository migration to `packages.broadcom.com`, and `salt-bootstrap` is non-functional as fixes to the scripts are underway.

- Details: https://saltproject.io/blog/salt-install-guide-overhaul/

**Comment by rratkiewicz** [comment]
Why weren't these changes coordinated prior to killing the old URL?  I'm questioning the viability of this project, as the previous repo migration was also not exactly well planned out or executed.
